### DISCUSSION TITLE
External links in new tabs

### DISCUSSION THREAD

**Discussion by alandefreitas** [squidfunk/mkdocs-material#3141] [Q&A] [answered]
Is there an easy way to open all external links in a new tab and internal links in the same tab?

**Comment by squidfunk** [comment]
You could use some [additional JavaScript](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript):

``` js
document.addEventListener("DOMContentLoaded", () => {
  for (const link of Array.from(document.querySelectorAll(".md-typeset a[href^=http]")))
    link.target = "_blank"
})
```

**Comment by squidfunk** [comment, marked as answer]
The [built-in privacy plugin](https://squidfunk.github.io/mkdocs-material/setup/ensuring-data-privacy/#built-in-privacy-plugin) now allows to automatically [add `target=_blank` to all external links](https://squidfunk.github.io/mkdocs-material/setup/ensuring-data-privacy/#+privacy.external_links_attr_map), so they open in new tabs.
